packages/editor/src/pages/Editor/store/header/selectors.ts

```diff
@@ -343,15 +343,34 @@ export const getFarItems = createSelector(
 );
 
 function shouldShowPopoutControls() {
-  // On desktop, show the popout control in Outlook -- but DON'T for Word/Excel/PPT,
-  //    since on those hosts, you can just resize the taskpane (or even drag it out!).
-  //    And relative to a popped-out taskpane, this has a few advantages:
+  // IMPORTANT: IF YOU MAKE ANY CHANGES HERE, UPDATE THE RUNNER'S
+  // "shouldShowPopoutControls" logic to be similar!
```

Comment:
Why not common then? Also, how come one has the getIsInAddin, and the other one doesn't?